### ISSUE TITLE
Add options for training parameters

### ISSUE THREAD

**Issue by sgrosser** [martinjm97/ENNUI#18]
Batch size, learning rate, optimizers (Adam, Momentum, etc), training and validation accuracy on graph. 
We can add even more for V2
